lib/AST/NameLookup.cpp

```diff
@@ -28,6 +28,11 @@
 #include "llvm/ADT/DenseMap.h"
 #include "llvm/ADT/TinyPtrVector.h"
 
+#if defined(_MSC_VER) && !defined(__clang__)
```

Comment:
Ah, both. :-) Please move `<cassert>` out of the `#if`, but don't bother asserting the popcount under MSVC (just `#if` it out).